clade/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "clade"
```

Comment:
Might as well keep calling it `clade`, though we might need to get more people involved in naming it if we ever want to spin it out into a separate open source project. (alternative name: simply `seafowl-metastore`)